seismic/xcorqc/correlator.py

```diff
@@ -354,11 +359,12 @@ def outputConfigParameters():
                                                      "single CC function, aimed at producing empirical Greens "
                                                      "functions for surface wave tomography.")
 @click.option('--restart', default=False, is_flag=True, help='Restart job')
+@click.option('--no-tracking-tag', default=False, is_flag=True, help='Do not tag output file names with a time-tag')
 def main(data_source1, data_source2, output_path, interval_seconds, window_seconds, resample_rate,
```

Comment:
At some point I suggest incorporating all these many options into an options class to encapsulate them.

Reply:
I've been thinking about reading them all from a yaml file..